.github/pull_request_template.md

```diff
@@ -0,0 +1,27 @@
+# [BUG | FEATUER | DOC] [TITRE]
+
+closes #[NUMERO_DU_TICKET_OU_LIEN]
+
+## Description
+
+Critères de succès
+
+## Validation
+
+- [ ] Code Approved
+- [ ] Test Coverage
```

Comment:
These two may be redundant. Github already offers the feature approve the code. And we use the Github actions for the "Test coverage".

Reply:
this is not the same, this is a flight check to make sure tests are added and valider by a person.